php/libraries/Database.class.inc

```diff
@@ -760,15 +760,15 @@ class Database
     }
 
     /**
-     * Runs a query as a prepared statement and returns the values of the first
-     * column in the select statement. If multiple columns are given, only the first
-     * will be returned.
+     * Runs a query as a prepared statement and returns the values of the one
```

Comment:
"the one"?

Reply:
does that sound weird?? maybe its just how i speak